scripts/setup/upgrade-postgres

```diff
@@ -43,12 +43,22 @@ pg_ctlcluster "$UPGRADE_FROM" main stop
     rm -rf "$TEMP_CONF_DIR"
 )
 
-pg_upgradecluster "$UPGRADE_FROM" main
+# Capture the output so we know where the path to the post-upgrade scripts is
+UPGRADE_LOG=$(mktemp)
+pg_upgradecluster "$UPGRADE_FROM" main --method=upgrade --link | tee "$UPGRADE_LOG"
+SCRIPTS_PATH=$(grep -o '/var/log/postgresql/pg_upgradecluster-10-12-main.*' "$UPGRADE_LOG")
```

Comment:
I'm guessing the `10` and `12` here will vary with the versions.

Reply:
Whoops -- good catch!